### DISCUSSION TITLE
Is it possible to use a custom domain or a custom data API in URLs?

### DISCUSSION THREAD

**Discussion by Madhan-Rommala** [Azure/data-api-builder#1437] [Q&A]
Custom Domain - Need to find the way to configure the custom domain to API URL. currently the REST API will be reachable via "**http://dns-name-label.location.azurecontainer.io:port/..**" but I wanted to add customer domain as per my wish in API url like 
"**http://organization-name.com/api/...**" and  I don't wanted to mention location-name , azurecontainer, port numbers  in my api url. 

Is there a workaround for this?

**Comment by seantleonard** [comment]
Hi @madhan-Rommala, Are you using an Azure service to host your container? You reference `azurecontainer.io`. Do the instructions in this Microsoft doc meet your requirements? https://learn.microsoft.com/en-us/azure/container-apps/custom-domains-certificates

You can then set a custom global REST endpoint prefix using the Data Api builder runtime configuration:
https://github.com/Azure/data-api-builder/blob/main/docs/configuration-file.md#rest
```json
"runtime": {
  "rest": {
    "path": "/<custom-rest-api-path>",
    "enabled": true
  }, ...
}
```

**Reply by Madhan-Rommala** [reply]
Thank you for your prompt response. I'll try this.
